pkg/config/containers.conf

```diff
@@ -146,9 +146,13 @@ default_sysctls = [
 #
 # ipcns = "private"
 
-# Flag tells container engine to whether to use container separation using
-# MAC(SELinux)labeling or not.
-# Flag is ignored on label disabled systems.
+# keyring tells the container engine whether to create
+# a kernel keyring for use within the container.
+# keyring = true
```

Comment:
nit: consistent formatting. probably only worth fixing if you're fixing something else

otherwise, LGTM once other stuff is addressed

```suggestion 
# a kernel keyring for use within the container.
#
# keyring = true
```